eng/common/performance/performance-setup.sh

```diff
@@ -17,13 +17,15 @@ kind="micro"
 llvm=false
 monointerpreter=false
 monoaot=false
+monoaot_path=
 run_categories="Libraries Runtime"
 csproj="src\benchmarks\micro\MicroBenchmarks.csproj"
 configurations="CompliationMode=$compilation_mode RunKind=$kind"
 run_from_perf_repo=false
 use_core_run=true
 use_baseline_core_run=true
 using_mono=false
+using_aot=false
```

Comment:
This appears unused currently. It's assigned but never read. If it's to be used what's the difference between this and `monoaot` being `true`?

Reply:
I started with this pattern and than abandoned that approach. I will remove this.